packers/zkp.go

```diff
@@ -218,12 +221,13 @@ func verifyAuthV2Sender(from string, pubSignals []string) error {
 
 func checkSender(from string, id *core.ID) error {
 
-	did, err := core.ParseDIDFromID(*id)
+	senderDID, err := core.ParseDIDFromID(*id)
```

Comment:
we need to convert `from` to id and compare big integers. Because not id's will be convertible to dids.